bin/regionEditor.ts

```diff
@@ -0,0 +1,27 @@
+import os from "os";
+import fs from "fs";
+import { PolusBuffer } from "../lib/util/polusBuffer";
+
+export class regionEditor {
```

Comment:
This file should actually be executable instead of just exposing a class, maybe parse CLI and write the file?

Reply:
by cli do you mean have an actual input interface and everything? cuz i can do that but just makin sure that's what yall want

Reply:
Yes, all files in `bin/` should be something a user can actually run an execute. Being able to do like `npm run add-server my-server-name 1.2.3.4` would be neat

Reply:
aight i think i did just that, there are two npm run commands:
`npm run set-region <region-name> <ip> <output-file (optional)>`
`npm run add-server <server-name> <ip> <port> <output-file (optional)>`